identity-service/src/routes/trackListens.js

```diff
@@ -436,4 +437,48 @@ module.exports = function (app) {
       listenMap
     })
   }))
+
+  /**
+   * Gets user listen records in bulk.
+   * 
+   * GET query parameters:
+   *  startTime (string) the start time to fetch listens from (in unix seconds format)
+   *  limit (number) the number of records to fetch from each user listens and anonymous listens
+   */
+  app.get('/listens/bulk', handleResponse(async (req, res) => {
+    let { startTime, limit } = req.query
```

Comment:
Not an issue, but for discprov getting the first startTime I'm assuming ot can just put something way in the past, but we could also let it be omitted and assume it's starting the from beginning. No preference either way though, but it might be nice? but I like it being required. Thought I'd mention the idea.

Reply:
I would prefer that the client (discovery service) be in charge of this aspect & making the API surface layer here as specific as possible. I get where the idea's coming from tho, if this was a endpoint we wanted to publicize to external clients, sensible defaults make sense